# Motor commands message
float32 cw_speed
float32 ccw_speed
uint32 angle
time ts